components/sup/src/main.rs

```diff
@@ -206,9 +218,6 @@ fn cli<'a, 'b>() -> App<'a, 'b> {
             (@arg PEER: --peer +takes_value +multiple
                 "The listen address of an initial peer (IP[:PORT])")
             (@arg PERMANENT_PEER: --("permanent-peer") -I "If this Supervisor is a permanent peer")
-            (@arg PEER_WATCH_FILE: --("peer-watch-file") +takes_value conflicts_with[peer]
```

Comment:
I think you've got a rebase issue here, removing this was probably not intended :)